core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -157,6 +163,14 @@ class TransactionsTest extends KafkaServerTestHarness {
     readCommittedConsumer.assignment.asScala.foreach { tp =>
       assertEquals(1L, readCommittedConsumer.position(tp))
     }
+
+    // undecided timestamps should not be searchable either
+    val offsetsForTimes = readCommittedConsumer.offsetsForTimes(Map(
+      tp1 -> new JLong(latestWrittenTimestamp),
+      tp2 -> new JLong(latestWrittenTimestamp)
```

Comment:
Nit: `latestWrittenTimestamp: JLong` is a little better in that it will use a cache of commonly used values.